### ISSUE TITLE
Atelier semaine du numérique centre culturel Koekelberg

### ISSUE THREAD

**Issue by DewiBrunet** [openfab-lab/openfab#211]
- [ ] ref to file .md or pullRequest #: .................................................  
*(à remplir quand l'issue est résolue)*
## contexte

Demande de Magali, Animatrice - Chargée de projets au Centre culturel de Berchem-Ste-Agathe/Koekelberg

"Je reviens donc vers toi concernant la possibilité de travailler ensemble lors de la semaine du numérique qui aura lieu cette année du 22 octobre au 2 novembre. 
Comme tu l'aura remarqué, les vacances de la Toussaint tombent dans cette période. Ici au Centre culturel, pendant les vacances scolaires, nous proposons des stages pour les enfants dont les parents travaillent. Pour la Toussaint on pensait proposer un stage de 3 jours (de 9h00 à 12h00 et de 13h00 à 16h00) et de l'axer sur le thème du numérique - avec un prisme assez ludique, bien entendu ! 
En général nous travaillons avec deux tranches d'age : les 3-5 ans et les 6-8 ans. Je crois que 3-5 ans c'est peut-être un peu jeune, mais penses-tu pouvoir proposer quelque chose pour la deuxième tranche d'âge ? 
Sinon, dans la période allant du 22 octobre au 2 novembre nous aimerions proposer des ateliers ponctuels s'adressant à des publics divers : ça peut être des familles, des seniors ou une expo pour le tout public... Bref, tout est encore à construire, donc si tu as des propositions elles sont plus que bienvenues ! :) 
Si tu veux on peut aussi en parler de vive voix, lors d'une réunion ou par téléphone !"

## proposition

- Atelier fablab mobile de 3 jours pour 6-8 ans? c'est pas trop jeune?

- du 22 octobre au 2 novembre : atelier? expo? vente? démo?

**Comment by zuperninja** [comment]
## objectif

Renvoyer une proposition avec devis

## ressources

Plein d'idée par ici : https://www.pinterest.fr/openfab/

## rewards

En crr ou vrai € (à discuter)

**Comment by DewiBrunet** [comment]
J'ai eu une relance la dessus, j'aimerai y répondre cette semaine mais :  

- Atelier fablab mobile de 3 jours pour 6-8 ans c'est pas trop jeune? Qu'est ce qu'on propose comme contenu à cet âge?
- Pour du 22 octobre au 2 novembre, que proposer : atelier? expo? vente? démo? Le plus facile serait p-e des ateliers mais combien, pour qui, avec quel contenu? Avec on peux aussi préparer un mini stand d'expo/vente de petit mobilier?

(je peux pas vraiment faire de devis sans ces infos... Anne-Ca, j'ai vu que tu as posté cette issue sur Facebook, merci, p-e est-ce difficile à la communauté de répondre à une demande très ouverte?)

**Comment by nicolasdb** [comment]
6-8 ans, je pense que c'est trop jeune pour fabmob. De mémoire, ça commence à 10ans jusque 18. 

Odile (CA) et Lara (madeinkit) sont à l'aise dans l'organisation de workshop pour cette tranche d'age. Elles peuvent peut-être conseiller, aider ou participer.

**Comment by DewiBrunet** [comment]
Ok, j'ai renvoyé vers tes contacts pour l'atelier jeune.

Et pour l'autre partie, Magali passe ce vendredi à 15h pour en discuter de vive voix avec moi.

**Comment by DewiBrunet** [comment]
Bilan du Rdv:

- proposer 2 ou 3 ateliers  de 3h pour des ados (15 max), à fixer avec FabMob pour du 29 octobre au 2 novembre
- mini expo d'objets/projets fait par des membres à déposer avec cartes de visites

Niveau contenu ça pourrait être : 
- atelier programmation avec les robots
- pins customisables avec la mini laser
- écussons textiles avec leds (sur le thème Halloween?)

Je vois avec Margot si c'est ok pour FabMob puis je confirme avec Magali avant fin juin pour fixer la communication (quelques lignes + photos pour mettre dans la gazette de la commune).

**Comment by nicolasdb** [comment]
- wearable, sandwish textile, découpe au laser, petite led et broderie
![image](https://user-images.githubusercontent.com/12049360/40848182-baf5a154-65be-11e8-9388-ffb7681e9103.png)
http://thecraftyavenger.com/electric-sugar-skulls/

**Comment by Anou92** [comment]
yes cool ça m'intéresserais d'animer un des ateliers en ce qui concerne la partie textile. On peut en parler cette après-midi, je vais venir travailler au fablab.

**Comment by DewiBrunet** [comment]
J'ai passé la balle à Margot pour la suite de l'organisation (pas fou l'abeille, je suis pas coordinateur^^)

Et j'ai eu Yorick au tel : 
- On peut pas faire de workshops payants sous Fablab Mobile, Patricia a déjà demandé à Innoviris et ils veulent pas :(  @nicolasdb bon à savoir pour les prochaines propositions qu'on peux faire!
- Atelier e-wearable textile ok c'est cool mais pas de budget à priori... je vais quand même essayé de le pousser lors de la réunion animateur du 20/06 car j'y vois un grand intérêt. Mais si ça se fait, le contenu sera sans doute fait par un salarié du projet qu'on peux mettre la dessus, désolé @Anou92 pour ces faux espoirs mais j'ai pas envie de te mettre sur un truc non payé (après comme je t'ais dis c'est un truc qui m’intéresse vachement, ça pourrait être un module qu'on développe nous à l'OpenFab ;) )

![but-are-my-money-wtf_o_3838635](https://user-images.githubusercontent.com/25649502/41116896-e47a8eaa-6a8b-11e8-9948-ff2c080a5ae5.jpg)

**Comment by Anou92** [comment]
Yes pas de soucis je comprends! ça pourrait être intéressant développer ça à OpenFab!

**Comment by nicolasdb** [comment]
Donc? On peut dire que c'est clos si le proj est transféré à fabmob?

**Comment by nicolasdb** [comment]
@DewiBrunet
Le lien est fait avec Fabmob c'est ça?
On cloture cette issue?

**Comment by nicolasdb** [comment]
- poke -
